Library/ViewModels/CommentsViewModel.swift

```diff
@@ -148,6 +151,12 @@ public final class CommentsViewModel: CommentsViewModelType,
         [comment.replyCount > 0, comment.status == .success].allSatisfy(isTrue)
       }
       .withLatestFrom(initialProject)
+
+    self.shouldShowLoadMoreIndicator = Signal
+      .combineLatest(isCloseToBottom.mapConst(true), self.isCommentsLoading)
```

Comment:
if you're mapping `isCloseToBottom` to true every time, is it needed to combine here?

Reply:
Yes, it's needed. `isCloseToBottom` ordinary returns void, so if it emits a value then I can map it as true. The condition is only satisfy if `isCloseToBottom` emits and `isCommentsLoading` emits value.

Reply:
I guess what I'm suggesting is that you probably don't need to be calling mapConst(true) here and checking for it's value because it will always be true. Something like this will probably give you the same result:

    self.shouldShowIndicator = Signal
      .combineLatest(isCloseToBottom, self.isCommentsLoading)
      .map { _, isCommentsLoading in isCommentsLoading }
      
      maybe using `takeWhen(isCloseToBottom)` would give the same desired effect too but I'm not 100 percent certain.

Reply:
@singhhari so I updated to this

```
self.shouldShowIndicator = Signal
  .combineLatest(isCloseToBottom, self.isCommentsLoading)
  .map { _, isCommentsLoading in isCommentsLoading }
```

But note that `isCloseToBottom.mapConst(true)` will only emit only when user scroll close to the bottom of the list. I have tested this with small and large comments.

Also `takeWhen(isCloseToBottom)` won't work in this case because you always need a `true` or `false` value.

Reply:
Yup sounds good. I just wanted to simplify it without compromising functionality.